main.py

```diff
@@ -1,10 +1,46 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_description in players.items():
+        race = Race.objects.get_or_create(
+            name=player_description["race"]["name"],
+            description=player_description["race"]["description"]
+        )[0]
+
+        if player_description["guild"]:
+            guild = Guild.objects.get_or_create(
+                name=player_description["guild"]["name"],
+                description=(
+                    player_description["guild"]["description"]
+                    if player_description["guild"]["description"]
+                    else None
+                )
+            )[0]
+        else:
+            guild = None
+
+        for skill in player_description["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        Player.objects.create(
```

Comment:
Consider using 'Player.objects.get_or_create()' instead of 'Player.objects.create()' to avoid creating duplicate player entries if the script is run multiple times with the same data.